### DISCUSSION TITLE
🤵🏻 Did anybody get married?

### DISCUSSION THREAD

**Discussion by DanielePucci** [robotology/community#357] [Announcements]
I would like to know if anybody in iCub married. Please help me.

**Comment by francesco-romano** [comment]
![EFAEED8C-F93E-4CF9-A934-ED9FC95DAE38](https://user-images.githubusercontent.com/2189180/60386719-2a306800-9a99-11e9-9eb8-dc292fdc0756.jpeg)

IMHO @traversaro

**Comment by DanielePucci** [comment]
Thank you very much @francesco-romano 

I think I got the  point.

Closing. 

If @traversaro doesn't agree, he can clearly open it again.

**Comment by lorejam** [comment]
@Silvio Traversaro <Silvio.Traversaro@iit.it> , what a commit(ment)!

Cheers :)
Lorenzo

On Sat, Jun 29, 2019 at 5:12 PM Daniele Pucci <notifications@github.com>
wrote:

> Thank you very much @francesco-romano
> <https://github.com/francesco-romano>
>
> I think I got the point.
>
> Closing.
>
> If @traversaro <https://github.com/traversaro> doesn't agree, he can
> clearly open it again.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/robotology/QA/issues/357?email_source=notifications&email_token=ABFOQN5IWEDUZCXZSSBXIK3P46CXRA5CNFSM4H4KVRIKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODY33URI#issuecomment-506968645>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABFOQN6KUVWL7LMZMZ4E5NLP46CXRANCNFSM4H4KVRIA>
> .
>


-- 
===================================================================
Lorenzo Jamone
Lecturer in Robotics
Turing Fellow - https://www.turing.ac.uk/people/researchers/lorenzo-jamone
ARQ - Advanced Robotics at Queen Mary - https://www.qmul.ac.uk/robotics/
School of Electronic Engineering and Computer Science -
https://www.eecs.qmul.ac.uk/
Queen Mary University of London - https://www.qmul.ac.uk/
https://lorejam.blogspot.com
l.jamone@qmul.ac.uk

Visiting Associate Researcher at VisLab (ISR/IST-Lisbon)
ljamone@isr.tecnico.ulisboa.pt
VisLab - http://vislab.isr.tecnico.ulisboa.pt
Institute for System and Robotics - https://welcome.isr.tecnico.ulisboa.pt/
Instituto Superior Técnico - https://tecnico.ulisboa.pt/

====================================================================
